bin/migrate-oats-data/applications/decisions/app_conditions/application_decision_conditions_init.py

```diff
@@ -0,0 +1,153 @@
+from common import (
+    OATS_ETL_USER,
+    setup_and_get_logger,
+    add_timezone_and_keep_date_part,
+    OatsToAlcsConditionTypeMapping,
+    BATCH_UPLOAD_SIZE
+)
+from db import inject_conn_pool
+from psycopg2.extras import RealDictCursor, execute_batch
+
+etl_name = "init_application_conditions"
+logger = setup_and_get_logger(etl_name)
+
+
+@inject_conn_pool
+def init_application_conditions(conn=None, batch_size=BATCH_UPLOAD_SIZE):
+    """
+    This function is responsible for initializing the application_conditions in ALCS.
```

Comment:
Nitpick: isn't this `application_decision_condition`?

Reply:
Correct, will fix